client/packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -93,32 +92,29 @@ export function Autocomplete<T>({
   inputProps,
   required,
   textSx,
-  createOption,
-  onCreateOptionClick,
+  clickableOption,
   ...restOfAutocompleteProps
 }: PropsWithChildren<AutocompleteProps<T>>): JSX.Element {
   const t = useTranslation();
   const openOverrides = useOpenStateWithKeyboard(restOfAutocompleteProps);
 
-  const isCreateOption = (option: unknown): option is AddOption => {
-    return (
-      typeof option === 'object' &&
-      option !== null &&
-      '_isAddOption' in option &&
-      (option as any)._isAddOption === true
-    );
+  const isClickableOption = (
+    option: unknown
+  ): option is ClickableOptionConfig => {
+    return option === clickableOption;
   };
 
-  const filter = useMemo(() => {
```

Comment:
Heh I was skeptical the useMemo would really have any benefit here so happy to see it go